gazelle/buf/generate.go

```diff
@@ -104,6 +116,14 @@ func generateBreakingRule(config *Config, target string) *rule.Rule {
 	return r
 }
 
+func generatePushRule() *rule.Rule {
+	r := rule.NewRule(pushRuleKind, "buf_push")
+	r.SetAttr("targets", []string{":buf"})
```

Comment:
Why this default, isn't it always overridden after resolving?

Reply:
Removing it